Resource.js

```diff
@@ -250,6 +253,7 @@ module.exports = function(app, route, modelName, model) {
             .exec(function(err, items) {
               if (err) return this.respond(res, 500, err);
               res.status(res.statusCode).json(items);
+              next()
```

Comment:
Sorry about the semicolons - bad habits from writing mainly in coffeescript.

I have also pushed up another change for your consideration. For the PUT, POST and DELETE methods I add the affected item to the `res.locals`. This has helped me out with my code as I need the item for socket signalling from the after handlers (see below). The change may be generally useful tho'

```
options.afterPut=(req, res, next)->
	sockets.emit 'PUT', item:res.locals.item
	next()
options.afterPost=(req, res, next)->
	sockets.emit 'POST', item:res.locals.item
	next()
options.afterDelete=(req, res, next)->
	sockets.emit 'DELETE', item: res.locals.item
	next()
```